### ISSUE TITLE
og:url contains the wrong URL

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Talk-archiver#59]
`og:url`, in the head of each page, is being published with the wrong URLs. Here are some examples:
https://talk.sciencegossip.org/recent
```
<meta property='og:url' content="https://talk.talk.sciencegossip.org/recent/" />
```
https://talk.higgshunters.org/recent/
```
<meta property='og:url' content="https://talk.higgs_hunter/recent/" />
```

This is a bug in the project code, which assumes that the Talk domain can be generated by swapping `talk.` for `www.` in the first build arg. However, the first build arg might be the Talk domain, or the name of an S3 bucket.

**Comment by eatyourgreens** [comment]
Chimp & See has no bucket path set, so we can't always rely on bucket path to give us the domain.
https://api.zooniverse.org/projects/chimp

**Comment by eatyourgreens** [comment]
Partially fixed by d80ccf5.

We still need to handle cases where we build by project name (because the bucket path is undefined or ambiguous
eg.
```
yarn build chimp
```
builds `talk.chimpandsee.org`.

Also cases where the bucket path does not map directly to the talk domain by prepending `talk.`
eg.
```
yarn build quench.galaxyzoo.org
```
builds `quenchtalk.galaxyzoo.org`.
